src/Microsoft.AspNet.Razor/CodeGenerators/Visitors/CSharpDesignTimeCodeVisitor.cs

```diff
@@ -98,6 +99,8 @@ private void VisitTagHelperDirectiveChunk(string text, Chunk chunk)
                 Writer.WriteVariableDeclaration("string", TagHelperDirectiveSyntaxHelper, "null");
             }
 
+            var text = ((Span)chunk.Association).Content.Trim();
```

Comment:
This doesn't work does it? What happens if the content has quotes? Wont the values come out double quoted?